sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -223,6 +223,11 @@ template <> class __SYCL_EXPORT command_graph<graph_state::executable> {
   std::shared_ptr<detail::exec_graph_impl> impl;
 };
 
+// additional deduction guide
```

Comment:
Would prefer this was styled the same as the other comments in the header.
```suggestion
/// Additional CTAD deduction guide.
```